client/src/components/Searchbar.js

```diff
@@ -15,14 +18,38 @@ const Input = styled.input`
   min-width: 284px;
   background-image: url(${SearchIcon});
   background-repeat: no-repeat;
-  background-position: 8px 7px;
+  background-position: 12px 10px;
   font: var(--font-robotolight);
 `;
 
-export default function Searchbar() {
+const ClearButton = styled.div`
+  position: absolute;
+  top: 57px;
+  right: 57px;
+  background: transparent;
+  border: none;
+  &:focus {
+    outline: none;
+  }
+`;
+
+export default function Searchbar({ onSubmit, onClear, ...props }) {
+  const handleClear = (e) => {
+    e.preventDefault();
+    console.log("onClear", onClear);
+    onClear();
```

Comment:
OnClear seems to work but also there will be a function as output. Maybe you could check it again?
![image](https://user-images.githubusercontent.com/68224104/95452248-417f2600-0969-11eb-92f8-efa4f6119242.png)